pkg/controller/container.go

```diff
@@ -244,8 +244,7 @@ func (r *Reconciler) Adjust(containerName string, memFactor float64, cpuFactor f
 		return fmt.Errorf("failed to patch container, want status code: %d, got %d",
 			http.StatusOK, resp.StatusCode)
 	}
-	r.L.Printf("patched container %s with mem factor: %.2f and new memory: %d bytes and with cpu factor : %.2f and new cpu: %dm.",
-		containerName, memFactor, newMemory, cpuFactor, newCPU)
+	r.L.Info().Str("container", containerName).Float64("memFactor", memFactor).Uint64("newMemory", newMemory).Float64("cpuFactor", cpuFactor).Uint64("newCPU", newCPU).Msg("Patched container")
```

Comment:
`Patched container` should be `patched container`. No caps in go comment is the most common.